h5io/_h5io.py

```diff
@@ -487,3 +499,31 @@ def list_file_contents(h5file):
         if not isinstance(h5file, h5py.File):
             raise TypeError(err.format(type(h5file)))
         _list_file_contents(h5file)
+
+
+def json_compatible(obj, slash='error'):
+    if isinstance(obj, (str, int, float, bool, type(None))):
+        return True
+    elif isinstance(obj, list):
```

Comment:
tuple ?

Reply:
The tuple is not supported in JSON https://en.wikipedia.org/wiki/JSON#Data_types.2C_syntax_and_example

Reply:
What we could do would be to pickle the object before storing it in HDF5 / JSON but then we get close to https://github.com/telegraphic/hickle and I guess that is a different focus.

Reply:
Agreed we should not go that route, thanks for the `tuple` clarification